.svlint.toml

```diff
@@ -0,0 +1,37 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+[option]
+exclude_paths = ["build.*", "sw/.*", ".sv.tpl$"]
+
+[rules]
+case_default = true
+enum_with_type = true
+for_with_begin = true
+function_same_as_system_function = false
+function_with_automatic = true
+generate_for_with_label = false
+generate_if_with_label = false
```

Comment:
this isn't matched to what we expect. The style guide expects the label starts with `gen_` but this rule only enforces the `if` or `for` statement to have label (any label). So that's why I disabled it but... yeah it is better than nothing. I will turn these on.